pkg/util/ovs/ovs.go

```diff
@@ -250,11 +250,19 @@ func (ovsif *ovsExec) Set(table, record string, values ...string) error {
 
 // Returns the given column of records that match the condition
 func (ovsif *ovsExec) Find(table, column, condition string) ([]string, error) {
-	output, err := ovsif.exec(OVS_VSCTL, "--no-heading", "--data=bare", "--columns="+column, "find", table, condition)
+	output, err := ovsif.exec(OVS_VSCTL, "--no-heading", "--columns="+column, "find", table, condition)
 	if err != nil {
 		return nil, err
 	}
-	return strings.Fields(output), nil
+	values := strings.Split(output, "\n\n")
+	// We want "bare" values for strings, but we can't pass --bare to ovs-vsctl because
+	// it breaks more complicated types
+	for i, val := range values {
+		if unquoted, err := strconv.Unquote(val); err == nil {
```

Comment:
Do we really want to ignore the error here?

Reply:
Yes. If `val` is a quoted string, then we know OVS would have quoted it correctly, so Unquote() won't return an error. So if Unquote() returns an error, that means the value wasn't a quoted string. (Eg, it was a number.)

Originally I had

    if val[0] == '"' && val[len(val)-1] == '"' {

before the Unquote call, but then that panics if val is empty (which it can be). And then while fixing it I realized that Unquote was going to do that check anyway so I should just let it do it.